### PR TITLE
add debouncing capabilities

### PR DESCRIPTION
**Pull request by JeandeCampredon**
The title explains pretty much the content.

I guess I am not the only one wanting it:
https://github.com/FaridSafi/react-native-google-places-autocomplete/issues/110

I haven't tested it much though.

Feel free to get back to me.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
   },
   "homepage": "https://github.com/FaridSafi/react-native-google-places-autocomplete#readme",
   "dependencies": {
+    "lodash": "^4.17.4",
```
Comment (myaskevich) [inline package.json]: Please replace it with `lodash.debounce` and `lodash.get` packages.

Comment (myaskevich) [inline package.json]: You can even drop `lodash.get` and do it manually ;)

Comment (JeandeCampredon) [inline package.json]: done 👍

**Comment by tusharkotlapure** [comment]
Hi @JeandeCampredon @FaridSafi ,

I feel defalut value for debaunce should be 0 user will set its value as required, and I found some errors in code i.e. comma(,) was missing which created problem while building.

**Comment by myaskevich** [comment]
Thanks @tusharkotlapure for the fix

**Comment by myaskevich** [comment]
Default `debounce` value will be `0` in `v1.2.10`. Thanks for your input @tusharkotlapure

**Pull request merged**